src/gui/graphdesigner.cpp

```diff
@@ -63,6 +63,11 @@ GraphDesigner::GraphDesigner(MainGUI* mainGUI, QWidget* parent)
     layout->setMargin(0);
     layout->setSpacing(0);
     layout->addWidget(m_innerWindow);
+    
+    m_curGraph->setViewInspector(parent->fullInspector());
+    connect(m_curGraph->view(), SIGNAL(nodeSelected(const Node&)), parent->fullInspector(), SLOT(slotSelectedNode(const Node&)));
```

Comment:
https://github.com/evoplex/evoplex/pull/44#discussion_r310407646
https://github.com/evoplex/evoplex/pull/44#discussion_r310407646
"the GraphDesignerPage could make the connection between them"